packages/api-solang/src/contractRpc.ts

```diff
@@ -79,18 +79,20 @@ function extractContractExecutionOutput(
     return { type: "success", value };
   } else {
     const dataView = new DataView(data.buffer);
-    const prefix = dataView.getUint32(0);
+    const prefix = data.buffer.byteLength >= 4 ? dataView.getUint32(0) : 0;
```

Comment:
I realized that a contract can revert with an empty output. I now handle this case.